chore: Adding ECS Fargate README (#393)
DataDog/datadog-cdk-constructs: src/ecs/fargate/README.md

3. **Feature Enablement**:

   - The construct provides granular control over Datadog features, such as:
     - **DogStatsD**: Enables custom metrics collection with configurable cardinality and socket support.
     - **APM**: Enables trace collection with optional Unix Domain Socket support.
     - **CWS**: Adds a security monitoring init container and wraps added container entrypoints.
     - **LogCollection**: Forwards logs to Datadog through the Fluentbit container.
   - These features are enabled or disabled based on the properties provided in the `DatadogECSFargateProps`.

4. **Seamless Integration**:
   - The `DatadogECSFargate` construct abstracts away the complexity of configuring Datadog monitoring for ECS Fargate tasks.
   - Developers can focus on defining their application containers, while the construct handles the creation and configuration of the Datadog Agent and related components.

## Testing

If you contribute to this package you can run the tests using `yarn test`. This package also includes a sample application for manual testing:

> 1. Open a seperate terminal.
> 2. Run `yarn watch`, this will ensure the Typescript files in the `src` directory are compiled to Javascript in the `lib` directory.
> 3. Navigate to `src/sample/ecs_fargate`, here you can edit `index.ts` to test your contributions manually.
> 4. At the root directory, run `npx cdk --app lib/sample/ecs_fargate/index.js <CDK Command>`, replacing `<CDK Command>` with common CDK commands like `synth`, `diff`, or `deploy`.

Reviewer: ```suggestion
1. Open a seperate terminal.
2. Run `yarn watch` to ensure the Typescript files in the `src` directory are compiled to Javascript in the `lib` directory.
3. Navigate to `src/sample/ecs_fargate` and edit `index.ts` to test your contributions manually.
4. At the root directory, run `npx cdk --app lib/sample/ecs_fargate/index.js <CDK Command>`, replacing `<CDK Command>` with common CDK commands like `synth`, `diff`, or `deploy`.
```

Author: Wording changes
